app/models/miq_worker.rb

```diff
@@ -370,8 +375,12 @@ def self.runner_script
     script
   end
 
+  def command_line
+    self.class.build_command_line(worker_options)
```

Comment:
If guid and ems_id are the only things we are using `worker_options` for, I think we should probably make this less opaque and pass those values explicitly.

The fact that `worker_options` is a hash makes it tough to see if we are covering all the cases for all the worker sub classes.

This can definitely be done in a follow up though.